src/Microsoft.Framework.Logging.Interfaces/ILoggerFactory.cs

```diff
@@ -9,6 +9,11 @@ namespace Microsoft.Framework.Logging
     public interface ILoggerFactory
     {
         /// <summary>
+        /// The minimum level of log messages sent to registered loggers
```

Comment:
Sorry, nit-picker's corner here - can you end the sentence with a period? :crying_cat_face: